src/main/java/com/github/pascalgn/jiracli/Jiracli.java

```diff
@@ -100,6 +106,15 @@ public static void main(String[] args) {
         map.put(Option.GUI, list.contains("-g") || list.contains("--gui"));
         map.put(Option.VERSION, list.contains("-V") || list.contains("--version"));
         list.removeAll(Arrays.asList("-h", "--help", "-c", "--console", "-g", "--gui", "-V", "--version"));
+        String line = null;
```

Comment:
I think this kind of "command with argument" parsing could be moved to a new method like `parseCommandArgument`

Reply:
Moved parsing of command line argument to the new method